static/css/main.css

```diff
@@ -19,22 +19,39 @@ h1 {
 }
 
 .org {
-  flex: 0 0 25%;
   margin-bottom: 1em;
+  width: 100%;
 }
 
 .org-image {
-  display: block;
-  width: 80%;
+  clear: none;
+  float: left;
+  width: 25%;
+}
+
+.org-image img {
+  width: 100%;
+}
+
+.org-info {
+  margin-left: 30%;
+  max-width: 75%;
+}
+
+.org-leaderboard li {
+  float: left;
+  width: 50%;
 }
 
-.none {
-  color: gray;
+img.github {
```

Comment:
Element (img.github) is overqualified, just use .github without element name. (overqualified-elements)

*Origin: CSSLintBear, Section: `css`.*